src/core/components/highlight-code.jsx

```diff
@@ -39,6 +39,16 @@ export default class HighlightCode extends Component {
     }
   }
 
+  componentDidMount() {
+    if (this.syntaxHighlighter) this.syntaxHighlighter.addEventListener("mousewheel", this.preventYScrollingBeyondElement, { passive: false })
```

Comment:
What would you say if we would reduce the code here a little bit? We have almost the same two lines of code here.

```js
[this.syntaxHighlighter, this.pre]
  .filter(element => !!element)
  .map(element => element.addEventListener("mousewheel", this.preventYScrollingBeyondElement, { passive: false }));
```

Reply:
thank you for the suggestion, I made the changes